### ISSUE TITLE
PKGBUILD for AUR

### ISSUE THREAD

**Issue by rharish101** [ssokolow/gtk-recent-scrubber#8]
The `gtk3` branch seems to work well for me, so I wrote a PKGBUILD for the AUR, since you've mentioned it in your TODO. Here's the file: https://bin.privacytools.io/?0b26f875fedff081#DedHoQUE3t7iHE8ADmB4HtsmQvhXyNtNsAJV5veVvQvc.

You just need to replace the maintainer's details with yours, test it out locally, and publish it to the AUR.

**Comment by ssokolow** [comment]
I appreciate the help but I've never used Archlinux (when I stopped using Gentoo, it was because I didn't have time to babysit the hiccups of a rolling release distro and switched to Lubuntu and Kubuntu LTS series) and, at the moment, I just don't have time to get familiar with AUR and take on responsibility for maintaining a PKGBUILD.

(If I did, I'd squash some of the bugs in the QuickTile repo.)

...but thanks for poking me about the gtk3 branch. I'll make another attempt to find some time to get it merged to master in the near future.

**Comment by rharish101** [comment]
That's understandable. In that case, would you be okay with me maintaining the PKGBUILD on the AUR? If you want, I can wait until you finish polishing the `gtk3` branch (and possibly merging to master).

FYI I've never maintained a package on the AUR before, but I've been using Arch Linux as my daily driver since 4 years ago.

**Comment by ssokolow** [comment]
Yeah, I'd be fine with that... but please do give me at least a few days to try to find time to finish checking the `gtk3` branch and move it to `master`.

I don't know if AUR is going to pull from the git repo but, if it will, I don't want to feel obligated to keep the `gtk3` branch in existence when it was intended to be just a temporary thing to get feedback on while I prepared to merge the changes to `master`. (Yeah, life hasn't been very cooperative over the last few years.)

**Comment by rharish101** [comment]
No problem, I'm not in a rush 😄. Take your time; I'll wait until you feel that it's good enough to publish it.

PS: If someone publishes it to the AUR, that doesn't mean that you have to maintain the `gtk3` branch forever. If you delete it, then it's the maintainer's job to update the branch the AUR PKGBUILD points to. Once that's done, no one would care about that branch, thanks to the rolling release model.
